CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to the CartoCSS map styles of OpenRailwayMap
+
+In this document, we provide a few tips if you want to contribute to the OpenRailwayMap CartoCSS map styles by creating pull requests.
+
+## Code formatting
+
+We use two spaces for indentation in our MapCSS and YAML files.
+
+## Test your code
+
+We like pull requests with example renderings. We will still make our own test renderings
+but your test renderings help you to find bugs. You can use the provided Docker setup.
+
+If you provide test renderings or not, we appreciate recommendations for locations where to
+make our test renderings.
+
+## New signalling systems
+
+Each country (sometimes each company) has their own signalling system. We welcome
+other systems than those already supported by the style. New signals must be documented
+on the OSM wiki and be actively used.
```

Comment:
maybe state in which wiki page/which language? (e.g. for germany it possibly would be https://wiki.openstreetmap.org/wiki/DE:OpenRailwayMap/Tagging_in_Germany and not a generic OSM-wiki page?)